core/src/WhiteRectDetector.cpp

```diff
@@ -276,29 +276,29 @@ bool WhiteRectDetector::Detect(const BitMatrix& image, int initSize, int x, int
 			return false;
 		}
 
-		ResultPoint x;
+		ResultPoint x_;
 		found = false;
```

Comment:
I think this sort of change reduces the readability of the code (at least if applied no matter what). I personally don't buy the argument that shadowing a variable is inherently bad. Scoping rules are clear and I would personally recommend to disable that warning.

Reply:
I agree that shadowing is not inherently bad, but not shadowing is not bad for sure :)
Maybe something more talking like `ResultPoint local_x` would be preferable.
Just my 2cents.

Reply:
I had a look at the surrounding code and if one really wanted to change the names here, I'd change x->l, z->b, y->r and also do that in the `CenterEdges` function. This way, they would actually convey some meaning (as in left, bottom, top, right).

Reply:
I just proposed the trailing addition because had no knowledge on what those variables meant.
If renaming is decided maybe is worth renaming them in a more meaningful way.
No problem for me to follow your suggestions.